transform/app/settings.py

```diff
@@ -108,8 +132,45 @@ def __init__(self, **kwargs):
                 "SEND_TO_SOLR or/and SEND_TO_S3 needs to be set to True. Otherwise, program will not have any effect"
             )
 
+        if self.CREATE_RELATIONS:
+            self.RELATIONS = self.get_relations_config()
+
         self.COLLECTIONS = self.get_collections_config()
 
+    def get_relations_config(self) -> dict:
+        """Get relations config"""
+
+        relations = {
+            self.TMP_DIRECTORY: {
+                self.DIRECTORY: "_tmp/",
+                self.COMBINED_PQ: "_tmp/combined_data.parquet",
+                self.ORGANIZATION_PQ: "_tmp/organization.parquet",
+                self.PROJECT_PQ: "_tmp/project.parquet",
+            },
+            self.DIRECTORIES_WITH_ADDITIONAL_COLUMNS: (self.ORGANISATION_PATH,),
```

Comment:
What wont change in the future I assume so it seems that self.ORGANISATION_PATH can be used directly in the code

Reply:
maybe some comment somewhere that organisations have additional cols

Reply:
I can directly use settings.ORGANISATION_PATH, and I've left space to include more directories, such as project titles, similar to those in organisations. This approach also ensures transparency regarding the additional columns in some directories.